src/dodal/beamlines/__init__.py

```diff
@@ -14,6 +14,10 @@
     "s03": "i03",
 }
 
+BEAMLINE_LAB_MAPPING: dict[str, str] = {
+    "i22": "p38",
+}
+
```

Comment:
Isn't this functionally the same as the _BEAMLINE_NAME_OVERRIDES? $BEAMLINE doesn't match up to expected dodal beamline

Reply:
it's not the same in the grand context of things